modules/extensions-support/src/main/java/org/mule/module/extension/internal/util/IntrospectionUtils.java

```diff
@@ -362,4 +371,35 @@ public static String getSourceName(Class<? extends Source> sourceType)
         return sourceType.getSimpleName();
     }
 
+    /**
+     * Returns the content of a field for a given annotation.
+     */
+    public static <T> T getAnnotationFieldValue(Element rootElement, Class<? extends Annotation> anAnnotation, String annotationField)
```

Comment:
for now I would place this method in the AnnotationProcessor that uses it. If we need it somewhere else, we can move it to a ASTUtils class